src/Pages/HomePage/componants/MainActivity.tsx

```diff
@@ -0,0 +1,24 @@
+import ActivityItem from './ActivityItem ';
+const MainActivity = () => {
+  return (
+    <>
+      <h6 style={{color: "#b5c1ce"}}>Today</h6>
+      <ActivityItem
+        name={'Supermarket'}
+        backgroundColor={'#ff0691'}
+        time={'10:36'}
+        amount={'12.45'}
+        icon={'fab fa-lyft'}
+      />
+      <ActivityItem
+        name={'H&M'}
```

Comment:
for string values, you can put them directly without curly braces{}
ex: `name='H&M'`